Gemfile

```diff
@@ -3,6 +3,7 @@ source "https://rubygems.org"
 gem "rake"
 gem 'wdm'
 gem 'bootstrap', '~> 4.0.0'
+gem 'armor'
```

Comment:
Why is this better that the build in ruby functions?

Reply:
Build-in functions have linear computational cost. [PBKDF2] (https://en.wikipedia.org/wiki/PBKDF2) was designed specifically for password hashing to avoid attacks based on computation timing.

Reply:
Cool